src/RoutePatches.tsx

```diff
@@ -92,7 +122,9 @@ export function patchLibraryPage(serverAPI: ServerAPI, storage: Storage): Mounta
 			props.children,
 			"type",
 			(_, ret1) => {
-				updatePlaytimes(storage)
+				//console.info('ret1', ret1);
+				//updatePlaytimes(storage, true);
+				//ret1.key = Math.random();
 				return ret1;
```

Comment:
So it looks like this patchers no longer needed, because commented code never executed, probably should patchers should be removed as well